shotgun_api3/shotgun.py

```diff
@@ -2250,14 +2256,16 @@ def share_thumbnail(self, entities, thumbnail_path=None, source_entity=None,
 
         result = self._send_form(url, params)
 
-        if not result.startswith("1"):
-            raise ShotgunError("Unable to share thumbnail: %s" % result)
-        else:
+        if result.startswith("1:"):
```

Comment:
I notice this changed from testing `startswith("1")` to `startswith("1:")`, but below we're just looking for `startswith("2")` -- should these two tests be consistent?

Reply:
Disregard this -- I just read your PR on the server-side code and see why this is.